### DISCUSSION TITLE
Issue with model evaluation in clam

### DISCUSSION THREAD

**Discussion by sreens** [jamesdolezal/slideflow#219] [Q&A]
I am attempting to evaluate a clam model trained for a multi-class classification (3 classes). However, I am getting the following error:

.conda/envs/slideflow/lib/python3.10/site-packages/slideflow/clam/utils/eval_utils.py", line 107, in summary
    if args.micro_average:
AttributeError: 'types.SimpleNamespace' object has no attribute 'micro_average'

This error has not occurred on previous runs (more than a month ago) but has appeared now. The version of slideflow I am using is: 1.4.0.post1 

If anyone has run into this issue, and know of a resolution, please advise. 
Thanks

**Comment by jamesdolezal** [comment]
Thanks for raising this! I will take a look later today. Could you make an issue (https://github.com/jamesdolezal/slideflow/issues) in the meantime? It's helpful for tracking in case other people have the same issue in the future.

**Reply by sreens** [reply]
Will do, thank you!
